### ISSUE TITLE
question about training

### ISSUE THREAD

**Issue by xiaozibuqi** [ChengxuLiu/MISCFilter#3]
The problem of gradient explosion happened when I trained on my dataset, there was no change in the parameters about learning rate, epochs, batchsize, etc., only patch_size was changed, have you encountered this problem?
